packages/browser/types/test/types.test.js

```diff
@@ -1,16 +1,12 @@
-const { describe, it, expect, afterAll } = global
+const { describe, it, expect } = global
 
 const { spawnSync } = require('child_process')
-const cleanupFiles = []
-
-afterAll(() => {
-  cleanupFiles.forEach(f => spawnSync('rm', ['-f', `${__dirname}/fixtures/${f}`]))
-})
 
 const assertTsProgramCompiles = p => {
-  cleanupFiles.push(`${p}.js`)
   const { stdout, stderr, err } = spawnSync('./node_modules/.bin/tsc', [
     '--strict',
+    '--noEmit',
```

Comment:
The [`--noEmit` flag](https://www.typescriptlang.org/docs/handbook/compiler-options.html) can be used to avoid having to emit files and clean up manually